core/services/feeds/orm.go

```diff
@@ -103,3 +106,79 @@ func (o *orm) CountManagers() (int64, error) {
 
 	return count, nil
 }
+
+// CreateJobProposal creates a job proposal.
+func (o *orm) CreateJobProposal(ctx context.Context, jp *JobProposal) (uint, error) {
+	var id uint
+	now := time.Now()
+
+	stmt := `
+		INSERT INTO job_proposals (spec, status, job_id, feeds_manager_id, created_at, updated_at)
+		VALUES (?, ?, ?, ?, ?, ?)
+		RETURNING id;
+	`
+
+	row := o.db.Raw(stmt, jp.Spec, jp.Status, jp.JobID, jp.FeedsManagerID, now, now).Row()
+	if row.Err() != nil {
+		return id, row.Err()
+	}
+
+	err := row.Scan(&id)
+	if err != nil {
+		return id, err
+	}
+
+	return id, err
+}
+
+// ListJobProposals lists all job proposals
+func (o *orm) ListJobProposals(ctx context.Context) ([]JobProposal, error) {
+	jps := []JobProposal{}
+	stmt := `
```

Comment:
Will this get large?

Reply:
This could get large, but it's an initial implementation. Will need to see what the frontend team need before expanding on this.